install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -123,6 +123,10 @@ spec:
           name: istio
       - name: istio-certs
         secret:
-          secretName: "istio.istio-pilot-service-account"
+{{- if .Values.global.rbacEnabled }}
+          secretName: istio.istio-pilot-service-account
+{{- else }}
+          secretName: istio.default
```

Comment:
Why using `istio.default`?

Reply:
Because pilot needs a mountPath (/etc/certs) to a secret volume otherwise it crashes. As explained in the description of this PR the istio.default secret is generated by the Citadel because the default ServiceAccount is there from K8s.